frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -98,38 +98,74 @@ const pageDescription =
   'Catalog. Cluster admins can install additional apps which will show up here automatically.';
 
 // Filter property white list
-const filterGroups = ['kind'];
+const filterGroups = ['type'];
 
 const getAvailableFilters = (initialFilters) => {
   const filters = _.cloneDeep(initialFilters);
-  filters.kind = {
-    ClusterServiceClass: {
-      label: 'Service Class',
-      value: 'ClusterServiceClass',
+  filters.type = {
```

Comment:
I do not get this change? Why was this necessary? The filters worked out previously as intended even without these changes.

Reply:
The changes here also broke the badge feature. Now there are no badges being shown on the card.

Reply:
filterGroup has now been changed to `type` instead of `kind`. The ability of a `type` is to hold more than one `kind` wherever required.

Reply:
The badges work perfectly with the latest changes

Reply:
I don't see any badges in the screencast that you've shared with the PR.

Reply:
Do you have an example type right now that has more kinds in it?

Reply:
No, but the initial plan was to put in unmanged services as a `type` with `imageStream`, `templates` & `ServiceClass`. I wouldn't see a change from the filterGroup `kind` if there wasn't a need for a type as such :)

Reply:
Updated the mockups. The badges are there.

Reply:
But i don't see a change where umanaged services are grouped with kinds `imageStream`, `template` and `serviceClass`. They are all a separate type right now.

Reply:
The way I see it right now is that all this change is unnecessary. The same could be achieved just by changing the name `Installed operators` -> `Operator Backed`, changing the order of the filters and just default the `active: true` for operator backed filter.

Reply:
Plus, i don't see the point of grouping multiple `kinds` into a `type` for filters. In the end it just needs to filter the items in catalog with ability to have multiple filters selected at once. That was already working perfectly fine whichout these changes. Am not sure if am missing something else here. That's why i added comments to the story as well.

Reply:
that is true. But this adheres to the initial discussion regarding the implementation. The changes can be reverted, if UX confirms that there is no need for grouping of the `kinds` in the near future.  Anyway I'll make the changes go away in that case.

Reply:
I just want to understand from an implementation point of view what benefits we are getting by grouping kinds here. UX may be thinking it in a way that all unmanaged service are in one group but purely from implementation and code changes what will be the benefits?

Reply:
From a flat data structure you're making it into a nested object struture. And then making the changes to the filtering logic to accomodate the change in data structure. Plus there is no actual grouping of `kinds` happening here. That doesn't seem right to me.

Reply:
The existing data structure used for the filters is neither a flat, nor a linear one. It comprises a single layer nesting of objects by `filterGroups`. For adding flexibility of having multiple kinds per `key` in a `filterGroup`, I simply added another layer of objects by nesting the same existing structure of data, which I referred to as grouping. However, this has been reverted.